.gitignore

```diff
@@ -148,3 +148,6 @@ cython_debug/
 
 # Intellig PyCharm
 .idea/
+
+# GitGuardian
+.cache_ggshield
```

Comment:
Why is this file always in a commit?

Reply:
Because the PR is not merged after the changes has been made to this file.